apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -19,6 +19,10 @@ export default class GameLab extends P5Lab {
     return P5LabType.GAMELAB;
   }
 
+  saveToProjectGallery() {
+    return false;
+  }
+
```

Comment:
Why do we need this and below for SpriteLab?  I think those project types can and should be publishable to the public gallery; if the saving to gallery here is a different gallery that I'm unaware of, can we specify which gallery in the function name please? Something like `saveToPersonalGallery` or `saveToSectionGallery`.

Reply:
I'm definitely looking for guidance here.  My goal was to modify the sharing behaviour for the two new poetry-based tutorials without changing behaviour for any other levels in our system.  

If I understand things right, Poetry inherits from SpriteLab which inherits from P5Lab.  But I think there can also be other activities in our system which use SpriteLab and GameLab directly, and those are the ones I particularly didn't want to change in any way.

cc @ajpal

Reply:
I've added some additional comments to the code to help explain what these are for.

Reply:
it seems like this is the default behavior we want (saveToProjectGallery returns false), so this function should probably live in P5Lab.js and only labs like Poetry that want to enable this will need to override it

Reply:
Good call.  Updated.